src/Pmi/Drc/RdrHelper.php

```diff
@@ -54,10 +54,12 @@ public function getClient($resourceEndpoint = null)
             $endpoint = $this->endpoint;
         }
 
-        $client = new \Memcache();
-        $cachePool = new MemcacheCachePool($client);
+        if (class_exists('\Memcache')) {
```

Comment:
Ran into this while testing locally, as later versions of PHP apparently do not bundle this support in.